source/meetings/2017/march/index.html.md

```diff
@@ -0,0 +1,102 @@
+---
+updated_by:
+  email: james@lazyatom.com
+  name: James Adam
+created_by:
+  email: james@lazyatom.com
+  name: James Adam
+category: meeting
+title: March 2017 Meeting
+updated_at:
+published_at: 2017-02-23 17:01:00 +0000
+created_at: 2017-02-23 17:01:00 +0000
+parts: {}
+status: Published
+---
+
+The March 2017 meeting of LRUG will be on *Monday the 13th of March*,
+from _6:00pm_ to _8:00pm_ (talks start at _6:30pm_).  The venue, [Code
+Node](https://skillsmatter.com/locations/264-skills-matter-codenode) between
+Moorgate and Liverpool St. stations, is provided by [Skills
+Matter](http://www.skillsmatter.com).  [Full venue and registration details are
+given below](#mar17registration).
+
+Agenda
+------
+
+After the flurry of February's traditional lightning talks, we're back to the
+regular format, with two talks for your enjoyment:
+
+### Action Cable vs. MessageBus - Live updates tailored for Rails
+
+[Michal Horčic](https://twitter.com/mousse85)
+
+> Comparison between Action Cable and MessageBus for live updates in Rails
+> application. Introduction to both solutions, basic implementation and benchmarks.
+
+### The story of the not-so dark magic of application preloading
+
+[Philipp Fehre](https://twitter.com/ischi)
+
+> Many of us use application pre-loaders like zeus or spring, but how do they
+> actually work? Lets step through what is actually going on by the example of zeus.
+
+### Dependency Injection with Little Boxes
+
+[Manuel Morales](https://twitter.com/manuelmorales)
+
+> A brief description of the benefits of dependency injection and how to gain
+> them with Little Boxes.
+
+
+Afterwards
+----------
+
+We aim to finish up the talks and formal part of the meeting by 8pm, but that
+doesn't mean you have to go home.  There's bound to be something to talk about
+after all those talks and we have two options for winding down the evening and
+chatting to other LRUG attendees:
```

Comment:
I didn't spend 8 years coming up with a different way to write this paragraph each time so you could just copy the previous one.... 😠 (probably don't want "all those talks" when there's only 3 rather than last months' 8.